src/main/java/app/linguistai/bmvp/utils/FileUtils.java

```diff
@@ -0,0 +1,18 @@
+package app.linguistai.bmvp.utils;
+
+import app.linguistai.bmvp.request.wordbank.QPredefinedWordList;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import java.io.InputStream;
+
+public class FileUtils {
+
+    public static QPredefinedWordList readPredefinedWordListFromYamlFile(String filePath) throws Exception {
```

Comment:
For loading quests from a yaml file, I used a different architecture (not in dev, in LINGUIST-...-Quest-... branch).

It looks like there should not be problems but have you tested that this works not only upon user microservice boot but also dynamically?

Reply:
The new default list is only added when registering for a new user, so yes.
Users created before this feature will not have a default word list